2_process/src/process_data.R

```diff
@@ -1,20 +1,12 @@
-
-library(dplyr)
-library(readr)
-library(stringr)
-library(sbtools)
-library(whisker)
-
 # Read in the data downloaded in the prior step
-read_eval_data = function() {
-  mendota_file = "1_fetch/out/model_RMSes.csv"
+read_eval_data = function(mendota_file = "1_fetch/out/model_RMSes.csv") {
```

Comment:
Just a note that the name of this function is a bit confusing, since when it is called we're actually reading in the raw data, which will then, once read, be processed to become `eval_data`. 

And a side note that we don't typically modularize actions as small as reading a csv into separate functions, though there's certainly no issue with it from the standpoint of how your code runs.

Reply:
Noted - I do see later you use this function to read in the `eval_data` csv. Probably best to rename this function and it's arguments to something more generic, or just include the `read_csv()` calls within the other functions.

Reply:
Thanks! I have removed it altogether in favor of just using the single-line `read_csv` call in the functions which need it. I was doing it to practice using a non-file output, and then kind of forgot about it!